src/index.html

```diff
@@ -1,16 +1,59 @@
 <!doctype html>
 <html lang="en">
+
   <head>
     <meta charset="UTF-8">
-    <meta
-      name="viewport"
-      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
-    >
+    <meta name="viewport"
+      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Search bar airbnb</title>
     <link rel="stylesheet" href="./style.css">
   </head>
+
   <body>
-    <h1>Search bar airbnb</h1>
+    <script type="text/javascript" src="./main.js"></script>
+    <form
+      action="#"
+      method="post"
+      role="search"
+      class="form"
+      data-qa="big"
+    >
+      <label>
+        <img
+          class="search-img search-img-big"
+          src="images/Search.svg"
+          alt="Search"
+        >
+        <input
+          data-qa="keypress"
+          id="tern-1"
+          type="text"
+          class="searchTern"
```

Comment:
1) use one type of cases
2) give more meaningful names
3) I'd recommend to use BEM convention to name classes. Like: 
`"form"`, `"form__input"`, `"form__image"`, `"form__image form__image--big"`
```suggestion
          class="input"
```